modules/op-using-custom-scc-to-improve-prod-security.adoc

```diff
@@ -0,0 +1,24 @@
+:_content-type: PROCEDURE
+
+[id='op-using-custom-scc-to-improve-prod-security_{context}']
+= Using a custom SCC to improve pod security
+
+Because you can migrate pipelines that must be run or built as root, you can improve security by using custom security context constraints (SCCs). To lock down your installation further when the default root user is not allowed, allow the container to run only as user ID `1000` when the pod requests what permissions to use.
+
+. Procedure
+
+* To set an SCC, edit the `pipelines-scc` and modify the `runAsUser` and `seLinuxContext` as follows:
```

Comment:
Minimalism:

Edit the `pipelines-scc` and modify the `runAsUser` and `seLinuxContext`.